### ISSUE TITLE
export to PDF crashes (Linux only)

### ISSUE THREAD

**Issue by gucio321** [musescore/MuseScore#17246]
### Issue type

Crash or freeze

### Bug description

When trying to export to PDF, everything crashes

### Steps to reproduce

- open any file
- click file -> Export
- click Export
- after a few seconds everything crashes with "core dumped"

### Screenshots/Screen recordings

<details><summary>logs</summary>
<p>

```
QML debugging is enabled. Only use this in a safe environment.
QSocketNotifier: Can only be used with threads started with QThread
20:25:34.639 | INFO  | main_thread | GlobalModule | onPreInit: log path: /home/mszeptuch/.local/share/MuseScore/MuseScore4Development/logs/MuseScore_230414_202534.log
20:25:34.639 | INFO  | main_thread | GlobalModule | onPreInit: === Started MuseScore 4.1.0-dev, build number  ===
20:25:34.660 | DEBUG | main_thread | DiagnosticsModule | onInit: crash server url: 
20:25:34.665 | INFO  | main_thread | DiagnosticsModule | onInit: success start crash handler
20:25:34.681 | DEBUG | main_thread | LinuxAudioDriver | open: Connected to default
20:25:34.681 | ERROR | main_thread | MuseSamplerLibHandler | MuseSamplerLibHandler: Unable to open MuseSampler library, path: /home/mszeptuch/.local/share/MuseSampler/lib/libMuseSamplerCoreLib.so
20:25:34.681 | ERROR | main_thread | MuseSamplerResolver | checkLibrary: Incompatible MuseSampler library; ignoring
20:25:34.681 | ERROR | main_thread | MuseSamplerLibHandler | MuseSamplerLibHandler: Unable to open MuseSampler library, path: libMuseSamplerCoreLib.so
20:25:34.681 | ERROR | main_thread | MuseSamplerResolver | checkLibrary: Incompatible MuseSampler library; ignoring
20:25:34.683 | DEBUG | main_thread | ShortcutsRegister | expandStandardKeys: removed 5 shortcut, because they are not bound to standard key
20:25:34.683 | DEBUG | main_thread | ShortcutsRegister | expandStandardKeys: added 3 shortcut, because they are alternative shortcuts for the given standard keys
20:25:34.685 | DEBUG | main_thread | ShortcutsRegister | expandStandardKeys: removed 176 shortcut, because they are not bound to standard key
20:25:35.074 | WARN  | main_thread | IpcSocket  | connect: failed connect to server
20:25:35.074 | WARN  | main_thread | IpcSocket  | connect: failed connect to server
20:25:35.074 | INFO  | main_thread | IpcSocket  | connect: success connected to ipc server
20:25:35.074 | INFO  | 139842810291904 | IpcServer  | listen: id: "cc628cf4dcc84a18b0270499acd6ff31" 
20:25:35.135 | DEBUG | main_thread | InstrumentTemplate | read: InstrumentTemplate:: init instrument <contrabass> not found
20:25:35.140 | DEBUG | main_thread | AlsaMidiInPort | connect: Connected to -1
20:25:35.140 | DEBUG | main_thread | AlsaMidiOutPort | connect: Connected to -1
20:25:35.156 | DEBUG | main_thread | Score      | tick2segment: no measure for tick -1
20:25:35.176 | DEBUG | main_thread | Score      | tick2segment: no measure for tick -1
20:25:35.183 | DEBUG | main_thread | WorkspacesDataProvider | rawData: get data from default workspace, key: ui_settings
20:25:35.183 | DEBUG | main_thread | WorkspacesDataProvider | rawData: get data from default workspace, key: ui_states
20:25:35.183 | DEBUG | main_thread | WorkspacesDataProvider | rawData: get data from default workspace, key: ui_toolconfigs
20:25:35.183 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:35.189 | DEBUG | main_thread | Score      | tick2segment: no measure for tick -1
20:25:35.201 | DEBUG | main_thread | Score      | tick2segment: no measure for tick -1
20:25:35.205 | DEBUG | main_thread | WorkspacesDataProvider | rawData: get data from default workspace, key: palettes
20:25:35.205 | DEBUG | main_thread | PaletteWorkspaceSetup | setup: there is palette data in the workspace, we will use it
20:25:35.388 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:35.469 | WARN  | main_thread | Qt         | DropArea: Dropping not implement for QtQuick on Wayland yet!
20:25:35.498 | ERROR | main_thread | App        | run: error: qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"

20:25:35.498 | WARN  | main_thread | Qt         | qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"
20:25:35.498 | ERROR | main_thread | App        | run: error: qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"

20:25:35.498 | WARN  | main_thread | Qt         | qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"
20:25:35.499 | ERROR | main_thread | App        | run: error: qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"

20:25:35.499 | WARN  | main_thread | Qt         | qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"
20:25:35.499 | ERROR | main_thread | App        | run: error: qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"

20:25:35.499 | WARN  | main_thread | Qt         | qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"
20:25:35.500 | ERROR | main_thread | App        | run: error: qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"

20:25:35.500 | WARN  | main_thread | Qt         | qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"
20:25:35.500 | ERROR | main_thread | App        | run: error: qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"

20:25:35.500 | WARN  | main_thread | Qt         | qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"
20:25:35.501 | ERROR | main_thread | App        | run: error: qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"

20:25:35.501 | WARN  | main_thread | Qt         | qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"
20:25:35.501 | ERROR | main_thread | App        | run: error: qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"

20:25:35.501 | WARN  | main_thread | Qt         | qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"
20:25:35.501 | ERROR | main_thread | App        | run: error: qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"

20:25:35.501 | WARN  | main_thread | Qt         | qrc:/qml/MuseScore/UiComponents/FlatButton.qml:174:5: QML Loader: Binding loop detected for property "itemImplicitWidth"
20:25:35.706 | WARN  | main_thread | AbstractMenuModel | makeMenuItem: not found action: tuning
20:25:35.757 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxHomeOpened
20:25:35.764 | WARN  | main_thread | Qt         | Wayland does not support QWindow::requestActivate()
20:25:35.764 | WARN  | main_thread | Qt         | QObject::connect(QQuickItem, mu::uicomponents::PopupViewCloseController): invalid nullptr parameter
20:25:35.764 | WARN  | main_thread | Qt         | QObject::connect(QObject, mu::uicomponents::PopupViewCloseController): invalid nullptr parameter
20:25:35.764 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxHomeOpened
20:25:35.765 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxHomeOpened
20:25:35.765 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxHomeOpened
20:25:35.867 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxHomeOpened
20:25:36.754 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxHomeOpened
20:25:36.843 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxHomeOpened
20:25:36.845 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxHomeOpened
20:25:36.845 | INFO  | main_thread | ActionsDispatcher | dispatch: try call action: file-open
20:25:36.864 | ERROR | main_thread | UpdateScenario | doCheckForUpdate: Unable to check for update, error: [1701] 
20:25:37.350 | DEBUG | main_thread | IpcChannel | syncRequestToAll: ret code: 1
20:25:37.350 | DEBUG | main_thread | NotationProject | load: try load: /home/mszeptuch/Documents/MuseScore4Development/Cloud Scores/Witaj_Polsko.mscz.autosave, format: mscz
20:25:37.389 | DEBUG | main_thread | XmlReader  | unknown: line 0 col 0: pedalBeginTextOffset
20:25:37.389 | DEBUG | main_thread | XmlReader  | unknown: line 0 col 0: ottavaTextAlign
20:25:37.391 | DEBUG | main_thread | XmlReader  | unknown: line 0 col 0: letRingBeginTextOffset
20:25:37.392 | DEBUG | main_thread | XmlReader  | unknown: line 0 col 0: palmMuteBeginTextOffset
20:25:37.392 | DEBUG | main_thread | XmlReader  | unknown: line 0 col 0: usePre_3_6_defaults
20:25:37.753 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxHomeOpened
20:25:37.760 | DEBUG | main_thread | WorkspacesDataProvider | setRawData: set data to default workspace, key: ui_states
20:25:37.827 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxHomeOpened
20:25:37.829 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxNotationFocused
20:25:37.991 | DEBUG | 139841197024960 | Score      | tick2segment: no measure for tick -1
20:25:38.033 | DEBUG | 139842851694272 | FluidSynth | init: synth inited

20:25:38.064 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.065 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.066 | DEBUG | 139842851694272 | FluidSynth | init: Selected preset 'Trumpet' on channel 0
20:25:38.079 | DEBUG | 139842851694272 | FluidSynth | init: synth inited

20:25:38.079 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.080 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.080 | DEBUG | 139842851694272 | FluidSynth | init: Selected preset 'Trumpet' on channel 0
20:25:38.088 | DEBUG | 139842851694272 | FluidSynth | init: synth inited

20:25:38.088 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.088 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.088 | DEBUG | 139842851694272 | FluidSynth | init: Selected preset 'Trumpet' on channel 0
20:25:38.096 | DEBUG | 139842851694272 | FluidSynth | init: synth inited

20:25:38.096 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.097 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.097 | DEBUG | 139842851694272 | FluidSynth | init: Selected preset 'Clarinet' on channel 0
20:25:38.110 | DEBUG | 139842851694272 | FluidSynth | init: synth inited

20:25:38.110 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.110 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.110 | DEBUG | 139842851694272 | FluidSynth | init: Selected preset 'Tenor Sax' on channel 0
20:25:38.139 | DEBUG | 139842851694272 | FluidSynth | init: synth inited

20:25:38.139 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.139 | INFO  | 139842851694272 | FluidSynth | addSoundFonts: success load soundfont: /usr/local/share/mscore-4.1/sound/MS Basic.sf3
20:25:38.140 | DEBUG | 139842851694272 | FluidSynth | init: Selected preset 'Woodblock' on channel 0
20:25:39.764 | WARN  | main_thread | Qt         | QQmlComponent: Created graphical object was not placed in the graphics scene.
20:25:39.769 | INFO  | main_thread | ActionsDispatcher | dispatch: try call action: nav-first-control
20:25:39.769 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxNotationFocused
20:25:39.851 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxNotationOpened
20:25:42.345 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxNotationOpened
20:25:42.354 | INFO  | main_thread | ActionsDispatcher | dispatch: try call action: file-export
20:25:42.386 | WARN  | main_thread | Qt         | Wayland does not support QWindow::requestActivate()
20:25:42.386 | WARN  | main_thread | Qt         | QObject::connect(QQuickItem, mu::uicomponents::PopupViewCloseController): invalid nullptr parameter
20:25:42.386 | WARN  | main_thread | Qt         | QObject::connect(QObject, mu::uicomponents::PopupViewCloseController): invalid nullptr parameter
20:25:42.386 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxNotationOpened
20:25:42.387 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:42.418 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxNotationOpened
20:25:42.501 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:43.953 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:44.313 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:44.649 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:44.997 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:45.348 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:46.948 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:47.694 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:48.688 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:48.690 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxNotationOpened
20:25:48.725 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxNotationOpened
20:25:48.727 | DEBUG | main_thread | UiActionsRegister | updateEnabledAll: currentCtx: UiCtxUnknown
20:25:48.761 | ERROR | main_thread | Qt         | Object 0x78dc9b0 destroyed while one of its QML signal handlers is in progress.
Most likely the object was deleted synchronously (use QObject::deleteLater() instead), or the application is running a nested event loop.
This behavior is NOT supported!
qrc:/qml/MuseScore/Project/ExportDialog.qml:172: function() { [native code] }
[16208:16208:20230414,202548.764373:WARNING thread_snapshot_linux.cc:111] Unknown scheduling policy 1073741826
[16208:16208:20230414,202548.766490:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.766666:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.766692:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.767584:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.767635:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768153:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768444:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768469:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768491:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768512:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768619:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768662:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768686:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768709:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768733:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768754:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768775:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768797:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768820:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768849:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768870:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768892:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768917:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768936:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768957:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.768975:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769013:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769031:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769084:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769123:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769141:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769158:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769242:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769260:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769311:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769332:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769349:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769369:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769388:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769412:ERROR elf_dynamic_array_reader.h:61] tag not found
[16208:16208:20230414,202548.769435:ERROR elf_dynamic_array_reader.h:61] tag not found
```

</p>
</details> 

<details><summary>neofetch output (system info)</summary>
<p>

```
OS: Fedora Linux 37 (Workstation Edition) x86_64 
Host: H610M S2H DDR4 -CF 
Kernel: 6.2.9-200.fc37.x86_64 
Uptime: 3 hours, 7 mins 
Packages: 3577 (rpm), 16 (flatpak) 
Shell: bash 5.2.15 
Resolution: 1920x1080 
DE: GNOME 43.4 
WM: Mutter 
WM Theme: Adwaita 
Theme: HighContrast [GTK2/3] 
Icons: HighContrast [GTK2/3] 
Terminal: gnome-terminal 
CPU: 12th Gen Intel i5-12600K (16) @ 4.900GHz 
GPU: Intel AlderLake-S GT1 
Memory: 6557MiB / 31856MiB 

```

</p>
</details> 

### MuseScore Version

1ddc8e66043ee0c25f5ca11cd26b6dcd66e19121 (master)

### Regression

Yes, this used to work in a previous version of Musescore 4.x

### Operating system

Fedora 37

**Comment by gucio321** [comment]
it used to work in v4.0.1 - will check newer version later

**Comment by cbjeukendrup** [comment]
Somewhat wild guess: it might be related to https://github.com/musescore/MuseScore/issues/17224, where this pull request is said to be the culprit: https://github.com/musescore/MuseScore/pull/16866

**Comment by gucio321** [comment]
> it used to work in v4.0.1 - will check newer version later

it still works in v1.0.2

**Comment by gucio321** [comment]
> Somewhat wild guess: it might be related to https://github.com/musescore/MuseScore/issues/17224, where this pull request is said to be the culprit: https://github.com/musescore/MuseScore/pull/16866

that's right - it broke since https://github.com/musescore/MuseScore/pull/16866

**Comment by DmitryArefiev** [comment]
Fixed in #17268

**Comment by gucio321** [comment]
Thank you! It made my life much easier :smile:

M. Sz. ***@***.***>

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
